content/zh/docs/kitex/Tutorials/code-gen/combine_service.md

```diff
@@ -1,12 +1,16 @@
 ---
-title: "Combine Service"
+title: "Combine Service [已弃用]"
 date: 2022-06-02
 weight: 2
 description: >
 ---
 
 > **本功能仅支持 Thrift 场景**
 
+**Combine Service 已经被弃用了。请尝试使用 [单 Server 多 Service](https://www.cloudwego.io/zh/docs/kitex/tutorials/advanced-feature/multi_service/) 功能。**
```

Comment:
Please use relative path links as we have multiple domain names.

Reply:
fixed.